fdbserver/ClusterRecovery.actor.cpp

```diff
@@ -778,6 +778,13 @@ ACTOR Future<Void> updateLogsValue(Reference<ClusterRecoveryData> self, Database
 	}
 }
 
+Future<Void> sendMasterNewTxnSystemInfo(ClusterRecoveryData* self, bool clear, RecruitFromConfigurationReply rep) {
+	RegisterNewInfoRequest req;
+	req.rep = rep;
+	req.clear = clear;
+	return brokenPromiseToNever(self->clusterController.registerNewInfo.getReply(req));
```

Comment:
Recovery is now driven by CC, so this is not needed since all actors in this files are run by CC.

Reply:
Yeah, but the CC and ClusterRecovery are still communicating these requests via the network. It's commented as a limitation in the code, I can plug in the change, but that it makes the change too large. 
Using this as a temporary solution.

Reply:
This change is still not bug-free on the test,
so many debugging symbols were added.
I will ask for your reviews later then it's correctness clean.